src/Windows/Avalonia.Win32/Win32Platform.cs

```diff
@@ -19,6 +19,7 @@
 using Avalonia.Utilities;
 using Avalonia.Win32.Input;
 using Avalonia.Win32.Interop;
+using JetBrains.Annotations;
```

Comment:
I want to remove JetBrains.Annotations at all from the repository. It can be and should be replaced with nullable attributes.
Out of scope of this PR tho.